public/components/sidebar.ejs

```diff
@@ -8,10 +8,10 @@
     data-bs-target="#testCollapse2"
     aria-expanded="false"
     aria-controls="testCollapse2"
-    disabled>Teams<i class="ms-auto bi bi-chevron-down text-light"></i></button>
+    >Teams<i class="ms-auto bi bi-chevron-down text-light"></i></button>
   <div class="collapse bg-light w-100" id="testCollapse2">
     <ul class="list-group">
-      <a href="/team" class="link-dark text-decoration-none">
+      <a href="/<%- firstName %>/" class="link-dark text-decoration-none">
```

Comment:
This `href` should be equal to what you set the `router` to point to. So, something like `/<%- firstName %>/teams` if navigating to a generic team page, or `/<%- firstName %>/<%- teamId>` if navigating to a specific team (which you would need to create another route path for.)